extension/js/common/api/key-server/attester.ts

```diff
@@ -103,12 +103,13 @@ export class Attester extends Api {
     return r.responseText;
   };
```

Comment:
Sorry to hijack this PR but I think this `updatePubkey` is probably not used? If it's not used, please remove the method. If used, please replace usage with either `submitPrimaryEmailPubkey` or `submitPubkeyWithConditionalEmailVerification`

Reply:
It's used here. 
But problem is this method passes `longId` instead of `email`. 
`submitPrimaryEmailPubkey` and `submitPubkeyWithConditionalEmailVerification` method doesn't seem to accept longId

https://github.com/FlowCrypt/flowcrypt-browser/blob/2baf09ee3bec3d7bcfae0a93c547f03be7548cff/extension/chrome/settings/modules/my_key_update.ts#L79

Reply:
Ah. Looks good.